spiders/test.py

```diff
@@ -1,15 +1,13 @@
 import scrapy
 
 
-class ProxiesDownloaderMiddleware(scrapy.Spider):
+class Test_spider(scrapy.Spider):
```

Comment:
Follow PEP-8 guidelines to name the class

Reply:
ok